gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ui/SnappyRegionStats.java

```diff
@@ -163,6 +193,9 @@ public void toData(final DataOutput out) throws IOException {
     toDataPre_STORE_1_6_2_0(out);
     out.writeInt(bucketCount);
     out.writeLong(sizeSpillToDisk);
+    out.writeInt(redundancy);
```

Comment:
Can you please investigate and tell why serializing this stats object is required.

Reply:
Yes,